include/nanorange/algorithm/stable_partition.hpp

```diff
@@ -16,13 +16,15 @@
 #ifndef NANORANGE_ALGORITHM_STABLE_PARTITION_HPP_INCLUDED
 #define NANORANGE_ALGORITHM_STABLE_PARTITION_HPP_INCLUDED
 
+#include "nanorange/iterator/operations.hpp"
```

Comment:
Angle brackets please. Also, please keep includes in alphabetical order. Is this header not already transitively included from somewhere?

Reply:
It is transitively included. This was just clangd automatically inserting includes and I didn't notice.